lib/gulp-image-converter.js

```diff
@@ -41,7 +42,29 @@ function convert(options, file, cb) {
   });
 }
 
+function optimizeSvg(svgo, file, cb) {
+  var data = file.contents.toString('utf8');
+  if (data.indexOf('&ns_') !== -1) {
```

Comment:
Oh, that's just the substring heuristic I mentioned [above](https://github.com/mozilla/teach.webmaker.org/pull/1052#issuecomment-113582058)... I've added a comment explaining things in the code!